cluster-up/cluster/kind/common.sh

```diff
@@ -145,26 +153,27 @@ function setup_kind() {
 
     _wait_kind_up
     _kubectl cluster-info
+    if [ $(_get_nodes |wc -l) -ge 2 ]; then
+        _fix_node_labels
```

Comment:
I would expect it to fix the labels all the time! Why only if >= 2. (asking as a casual user reading this for the first time) Could you move the if into the function? There if will be clearer why we need it. Also, you have to do some labeling even with a single node - mark the master with worker label

Reply:
Ok let's try to mark as workter too, the erro was with the taint since node is not a NonSchedule.